src/components/SelectedTapMobile/SelectedTapMobile.js

```diff
@@ -1,13 +1,29 @@
 import { Button, Collapse, IconButton } from '@mui/material';
 import { styled } from '@mui/material/styles';
-import React, { useEffect, useState } from 'react';
+import React, { useEffect, useRef, useState } from 'react';
 import styles from './SelectedTapMobileInfo.module.scss';
 
 import { ReactComponent as DirectionIcon } from '../images/ArrowElbowUpRight.svg';
 import { ReactComponent as CaretDownSvg } from '../images/CaretDown.svg';
 import { ReactComponent as ThreeDotSvg } from '../images/DotsThree.svg';
 import { ReactComponent as ExportSvg } from '../images/Export.svg';
 
+import FountainIcon from '../icons/CircleWaterIcon.svg';
+
+function getIconForTapType(tapType) {
+  switch (tapType) {
+    case 'Drinking Fountain':
+      return FountainIcon;
+    case 'Bottle filler and fountain':
+      return '/static/images/BottleFiller.png';
```

Comment:
Can we add these files to our src/assets directory - and import them here? So the build process will link these files

Reply:
This is strange... I feel like I copy-pasted this from somewhere, but now I can't even find where this came from, and those png files don't exist. 

In the meantime, I've removed those lines and just have the fountain icon, and I actually can't test the other icons because opening the resource selector on mobile results in an error. As part of my migration, I'll be reworking some of that data being passed in, so we have two options:

1. As part of this PR, leave it just as fountain icons, and then update the other icons in another PR
2. Try to fix the resource change panel opening. I may need to get some context from @gcardonag on this first, I started to go down a path to fix the issue (screenshot attached), but there are some interesting aspects of the v2 work and toolbar within desktop/mobile that make it tricky.

<img width="451" alt="Screenshot 2024-03-06 at 11 11 58 PM" src="https://github.com/phlask/phlask-map/assets/8053203/0c2ef8f2-0143-416f-8a2d-a621bdb72c8c">